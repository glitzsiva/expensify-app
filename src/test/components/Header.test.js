import React from 'react';
import Header from '../../component/Header';
import {shallow} from 'enzyme';

// import ReactShallowRenderer from 'react-test-renderer/shallow';

test('component Header render',()=>{

    const wrapper =shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    // const renderer= new ReactShallowRenderer();
    // renderer.render(<Header/>)
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});