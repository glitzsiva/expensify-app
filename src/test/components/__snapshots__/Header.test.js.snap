// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component Header render 1`] = `
<div>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
